pages/_app.page.tsx

```diff
@@ -69,14 +70,18 @@ function MyApp({
 
     pageProps.data.result = previewData;
 
+    const plausibleDomain = process.env.PLAUSIBLE_DOMAIN || "gieffektivt.no"; //TODO: Remove temporary fallback when Vercel setup is done
```

Comment:
Might be able to use existing `NEXT_PUBLIC_SITE_URL` env variable instead

Reply:
I will look into it! Yeah I believe the domain is not exactly the same as any of our existing variables due to the lack of HTTP protocol in front